File: HW1_VB/Form1.vb
Public Class Form1
    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Dim ComputerName As String
        ComputerName = System.Net.Dns.GetHostName
        RichTextBox1.Text = ComputerName
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim UserName As String
        UserName = SystemInformation.UserName
        RichTextBox1.Text = UserName
    End Sub

    Private Sub RichTextBox1_TextChanged(sender As Object, e As EventArgs) Handles RichTextBox1.TextChanged

    End Sub
End Class

File: HW1_CS/Form1.cs
using System.Net;

namespace HW1_CS
{
    public partial class Form1 : Form
    {
        string username = "";
        string hostname = "";
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.username = System.Security.Principal.WindowsIdentity.GetCurrent().Name;
            richTextBox1.Text = this.username + '\n';
        }

        private void button2_Click(object sender, EventArgs e)
        {
            this.hostname = Dns.GetHostName();
            richTextBox1.Text = this.hostname + '\n';
        }

        private void richTextBox1_TextChanged(object sender, EventArgs e)
        {

        }
    }
}